crates/wasi/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "wasmi_wasi"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+wasi-common = "2.0.1"
+wasi-cap-std-sync = "2.0.1"
+wiggle = "2.0.0"
+wasmi = { version = "0.20.0", path = "../wasmi" }
+wasmi_core = { version = "0.5.0", path = "../core"}
+
+[dev-dependencies]
+wat = "1.0.50"
+
+[features]
+default = ["std"]
+std = []
```

Comment:
It makes no sense for `wasmi_wasi` crate to support `no_std` at all since its entire functionality builds upon OS based features. So if a user uses `wasmi_wasi` they necessarily require an OS underlying to `wasmi` for all the stuff such as opening files etc.

Reply:
Note for example that Wasmtime's `wasi_common` crate also does not have an `std` crate feature and therefore also no `no_std` support. https://github.com/bytecodealliance/wasmtime/blob/0290a83502d3778d460a6ee131294603bd575efd/crates/wasi-common/Cargo.toml

Reply:
True! Thanks. Resolved